pkg/controller/build/build_controller.go

```diff
@@ -165,6 +175,13 @@ func (r *ReconcileBuild) Reconcile(request reconcile.Request) (reconcile.Result,
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileBuild) validateRuntime(runtime buildv1alpha1.Runtime) error {
+	if len(runtime.Directories) == 0 {
+		return fmt.Errorf("Runtime requires 'directories' to copy informed")
```

Comment:
Wording proposal: `The property 'spec.runtime.directories' must not be empty.`

Reply:
Thanks, sounds a lot better. :-)